pixart.js

```diff
@@ -1 +1,31 @@
+$("#form").submit(setColor);
 
+
+// Sets the color that user will paint with
+// ****************************************
+function setColor() {
+  event.preventDefault();
+  var $userInput = $("#color-field").val();
+  var $colorBox = $(".brush").eq(0);
+  $($colorBox).css("background-color", $userInput);
+}
+
+
+// Takes color from user and makes the divs bground that color
+// ***********************************************************
+function squareColor() {
+  var $userInput = $("#color-field").val();
```

Comment:
Technically, reading input from here can cause a bug when you edit the text box but don't submit. In this case, you're gonna paint the square the new color (even though they haven't clicked to set it).

The solution probably involves setting a variable that's shared between setColor and SquareColor...